templates/v2/user_creation/components/_password_card.html.twig

```diff
@@ -2,19 +2,12 @@
 
 {% block customContent %}
     <div class="text-start">
-        {% for field in form %}
-            {#          The 'plainPassword' field name is only used when creating a professionnal #}
-            {% if field.vars.name == 'password' %}
-                {{ include('v2/user/strong_password/_strong_password_row.html.twig', {
-                    'field': form.password,
-                    'isBeneficiaire': true,
-                    'hideAndShow': false,
-                    'generatePassword': true,
-                }) }}
-            {% else %}
-                {{ form_row(field) }}
-            {% endif %}
-        {% endfor %}
+        {{ include('v2/user/strong_password/_strong_password_row.html.twig', {
+            'field': form.password,
+            'isBeneficiaire': true,
+            'hideAndShow': false,
+            'generatePassword': true,
+        }) }}
         {{ include('v2/common/_form_actions.html.twig') }}
```

Comment:
I think we need this to display mfa fields at beneficiary creation

Reply:
"this" means the deleted code on this template

Reply:
wooops